tests/e2e/e2e-plugin/rest-api/namespace.php

```diff
@@ -41,6 +41,14 @@ function rest_route() : void {
 					delete_option( 'site_icon' );
 					update_option( 'blogdescription', 'Astra Test Enviornment' );
 
+					$all_users = get_users();
+					require_once(ABSPATH.'wp-admin/includes/user.php');
```

Comment:
In a separate PR maybe, enable phpcs in this directory - 
```suggestion
					require_once( ABSPATH . 'wp-admin/includes/user.php' );
```

Reply:
@Nikschavan Updated!